HashTagTokenizer.java

```diff
@@ -7,20 +7,32 @@ public static void main(String[] args) {
 		String hashTag = args[0];
 		String []dictionary = readDictionary("dictionary.txt");
 		breakHashTag(hashTag, dictionary);
+		
 	}
 
 	public static String[] readDictionary(String fileName) {
 		String[] dictionary = new String[3000];
-
+		String word = ""; 
 		In in = new In(fileName);
 
+		for (int i=0; i < 3000; i++){
+			word = in.readLine(); 
+			dictionary[i] = word; 
+		}
 		// Your code here
 
 		return dictionary;
 	}
 
 	public static boolean existInDictionary(String word, String []dictionary) {
-		// Your code here
+
+			for (int i=0; i<3000; i++){
```

Comment:
This method should support `unknown` length of dictionary, you should done it like this:
```suggestion
			for (int i=0; I<dictionary.length; i++){
```